content/report_RHF.tex

```diff
@@ -1,9 +1,244 @@
+    our first guess of the actual molecular orbitals. However, this guess will not be
+    very good yet, since off course the core Hamiltonian is not the same as the 
+    molecular Hamiltonian. 
+    \item A basis set is often chosen for the ease of integration. Often, these
+    are Gaussian functions. So we can see that the chosen basis set does not
+    need to have the same properties as the eigenfunctions of the Hamiltonian. 
+    Off course the functions do have to be "well-behaved". The size however, will always be the same.
+    If you choose a basis set with only two functions, your Hamiltonian will have only
+    two eigenfunctions. The bigger the basis set, the more eigenfunctions you will have. 
+    \item We are using an expansion of a function in a basis. That is always possible.
+    However, there is always a catch, namely that the basis set used has to be complete.
+    In most cases this would actually mean that it is infinitely large, so of no
+    real use to us. However, if we choose an incomplete set, we will only be able
+    to make approximations.
+    \item If we want to do the expansion, we need to integrate the spin out first.
+    So we need to end up with an expansion that is only dependant on the spacial
+    orbital, like in Equation \ref{eq:expansion}.
+    \begin{equation}\label{eq:expansion}
+        \psi_i = \sum_jC_{ji}\phi_j
+    \end{equation}
+\end{enumerate}
+
+\subsection{The Density Matrix}
+\label{subsec:theDM}
+At this point we still need to adress one topic, the density matrix. This will be
+very important since it will be used to calculate the Fock matrix in our program.
+To demonstrate what it is, we will look at the expression for the coulomb operator.
+When we take Equation \ref{eq:coulomb} and then expand the functions $\chi_i$ in
+a basis after eliminating the spin, we find Equation \ref{eq:coulombexp}.
+
+\begin{equation}\label{eq:coulombexp}
+    \hat{J}_i = \int\sum_{\sigma}C^*_{\sigma}\phi^*_{\sigma}(x_j)\frac{1}{|\boldsymbol{r}_1 - \boldsymbol{r}_j|}\sum_{\nu}C_{\nu}\phi_{\nu}(x_j)dx_j
+\end{equation}
+
+The C factors in this equation are merely the exapansion coëficcients of the function
+$\psi_i$.(Rememmber that this is the spatial orbital, see Equation \ref{eq:spinor})
+These are constans, so we can safely extract them from the integrals to form
+Equation \ref{eq:coulombexp1}.
+
+\begin{equation}\label{eq:coulombexp1}
+    \hat{J}_i = \sum_{\sigma}\sum_{\nu}C_{\nu j}C^*_{\sigma j}\int\phi^*_{\sigma}(x_j)\frac{1}{|\boldsymbol{r}_1 - \boldsymbol{r}_j|}\phi_{\nu}(x_j)dx_j
+\end{equation}
+We can apply this same strategy to the exchange operator. In the end we will be
+able to factor out the products of the expansion coëficcients and afer summation
+over all basis functions we can define the density matrix as Equation 
+\ref{eq:densitymatrix}.
+
+\begin{equation}\label{eq:densitymatrix}
+    D_{\sigma\nu} = 2\sum^{N/2}_jC_{\sigma j}^*C_{\nu j}

```

Comment:
```suggestion
    D_{\sigma\nu} = 2\sum^{N/2}_jC_{\sigma j}C_{\nu j}^*

```
In (almost) all sources the C matrix of the second index gets the conjugate, however this is a little arbitrary.